utils/zpe-updater/cmd/tools/main.go

```diff
@@ -87,7 +90,36 @@ func main() {
 		zpuConfig.Zts = ztsURL
 	}
 
-	// first check if we're just asked to view a local domain
+	// first, if running check we need to verify policy files
+	// validity and generate a json metrics that can be pushed to
+	// a monitoring service
+	if check {
+		var bytes []byte
+		policyStatus, errorMessages := zpu.CheckState(zpuConfig)
+		policyMetrics := metrics.FormPolicyMetrics(policyStatus)
+		for _, policyMetric := range policyMetrics {
+			metricBytes, err := metrics.DumpMetric(policyMetric, nil)
+			if err != nil {
+				errorMessages = append(errorMessages, err)
+			}
+			bytes = append(bytes, metricBytes...)
+		}
+		statusBytes, ok, err := metrics.DumpStatus(errconv.Reduce(errorMessages))
+		if err != nil {
+			byteString := fmt.Sprintf("{\"application\":\"zpu-check\",\"status_code\":1,\"status_msg\":\"%s\"}", err.Error())
```

Comment:
rather than having the json hardcoded here, let's move this into a method in the metrics file. This way the definition is in the same file so we know what we're generating something like:

byteString := metrics.GetFailedStatus(err.Error())

Reply:
fixed